java-source/src/test/java/com/example/contract/IOUFlowTests.java

```diff
@@ -41,44 +44,34 @@ public void tearDown() {
         net.stopNodes();
     }
 
-    @Test
-    public void flowRejectsInvalidIOUs() throws InterruptedException {
+    @Test(expected=ExecutionException.class)
```

Comment:
Because the current structure is very confusing. Java has no "failsWith()" method, so you have to do a try/catch, catch the exception, then confirm its type. You fail the test by NOT throwing an exception, which is very confusing to people.